src/site/content/en/blog/prefers-reduced-motion/index.md

```diff
@@ -171,29 +171,69 @@ mediaQuery.addEventListener('change', () => {
 Note that the parentheses around the actual media query are obligatory:
 
 {% Compare 'worse' %}
+
 ```js
-window.matchMedia('prefers-reduced-motion: reduce')
+window.matchMedia('prefers-reduced-motion: reduce');
 ```
+
 {% endCompare %}
 
 {% Compare 'better' %}
+
 ```js
-window.matchMedia('(prefers-reduced-motion: reduce)')
+window.matchMedia('(prefers-reduced-motion: reduce)');
 ```
+
 {% endCompare %}
 
+## Working with the media query from `<picture>` contexts
+
+An interesting use case is to make playing of an animated AVIF, WebP, or GIF dependent on the
+`media` attribute. If `(prefers-reduced-motion: no-preference)` evaluates to `true`, it's safe to
+display the animated version, else the static version:
+
+```html
+<picture>
+  <!-- Animated versions. -->
+  <source
+    srcset="nyancat.avifs"
+    type="image/avif"
+    width="250"
+    height="250"
```

Comment:
No need for these since they’re the same as the `<img>` dimensions:

```suggestion
```

Reply:
Even if the physical size is different?